src/main/java/duke/main/Duke.java

```diff
@@ -0,0 +1,46 @@
+package duke.main;
+
+import duke.command.*;
+import duke.exception.*;
```

Comment:
Would listing all the imported classes explictly make the class easier to maintain in the future?

i.e.
import duke.command.AddCommand
import duke.command.Command
...